app/commands.py

```diff
@@ -108,7 +108,7 @@ def wrapper(*args, **kwargs):
         return wrapper
 
 
-@notify_command()
+@notify_command("delete-functional-test-data")
```

Comment:
we dont need to add this arg - flask click commands can be called with the function name (with hyphens) - so we can just run `flask command purge-functional-test-data -u notify-test-preview`